static/scss/main.scss

```diff
@@ -217,48 +219,129 @@ a#skipnav {
 
 #home {
   .hero {
-    @include span-columns(8);
-    @include shift(2);
-
     text-align: center;
     color: white;
-    padding-top: 40px;
+    padding: 40px 0 60px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `40px 0 60px` instead of `40px 0 60px 0`